src/components/preview/components/page-preview-button.tsx

```diff
@@ -40,19 +44,32 @@ class PagePreviewButton extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
-      loading: false
+      mouseOver: false
     };
   }
 
   /**
    * Render
    */
   public render() {
+    const { view, selectedView, onResize } = this.props;
+    const { mouseOver } = this.state;
+    const selected = selectedView?.id === view.id;
+    const selectedHighlight = mouseOver || selected ?
```

Comment:
Could this be moved to styles? Or at lest generalized in one place, it's currently repeated in all of the components

Reply:
boxShadow definition moved to styles